xds/internal/xdsclient/xdsresource/filter_chain.go

```diff
@@ -112,7 +112,7 @@ func (f *FilterChain) convertVirtualHost(virtualHost *VirtualHost) (VirtualHostW
 	rs := make([]RouteWithInterceptors, len(virtualHost.Routes))
 	for i, r := range virtualHost.Routes {
 		var err error
-		rs[i].RouteAction = r.RouteAction
+		rs[i].RouteAction = r.ActionType
```

Comment:
Can you also change this while you're at it?

Reply:
Done.